.rubocop.yml

```diff
@@ -23,6 +23,7 @@ Metrics/ClassLength:
   CountAsOne: ['array', 'heredoc', 'hash']
   Exclude: 
     - app/models/service_day.rb
+    - app/controllers/api/v1/users_controller.rb
```

Comment:
Why is this file being excluded?

Reply:
The Rubocop error is for Class lengths that are over 100 lines and the changes to users_controller made it over 100 lines(101 lines).  I just pushed up a commit to make the filter_date method a ternary operator to get it under the 100 lines.  Another option is I could move the filter date and the dashboard_query methods into a service_object if preferred.